sdk/tests/conformance/extensions/oes-texture-float.html

```diff
@@ -114,24 +114,20 @@
 
         (function() {
             debug("");
+            var renderable = isRenderable(gl);
             var renderableExtName = "WEBGL_color_buffer_float";
-            if (isRenderable(gl)) {
-                var supported = gl.getSupportedExtensions();
-                assertMsg(supported.includes(renderableExtName),
-                          "Implicit renderability requires exposing support for " + renderableExtName);
-            } else {
-                var renderableExt = gl.getExtension(renderableExtName);
-                if (!renderableExt) {
-                    testPassed(renderableExtName + " not supported. This is fine.");
-                    return;
-                }
-                testPassed(renderableExtName + " supported. Proceeding to its tests.");
+            var supported = gl.getSupportedExtensions().includes(renderableExtName);
+            if (renderable && !supported) {
+                testFailed("RGBA/FLOAT is color renderable but " + renderableExtName + " not exposed");
+            } else if (supported && !renderable) {
+                testFailed(renderableExtName + " is exposed but RGBA/FLOAT is not color renderable");
+            }
+            if (supported) {
```

Comment:
This should be run even if it's not claimed-supported if it is (incorrectly) implicitly renderable.
You introduce a bug here by not calling getExtension(), which means this will rely on implicit activation. IIRC the idea is that we drop recommendation that new implementations require explicit activation here, but I think we stopped short of requiring implicit activation. If we change this we *must* change the specs to reflect this.

Reply:
I thought the idea was that the current change to the OES_texture_float and OES_texture_half_float extension specs removing the sentence "Although this feature has historically been allowed..." mandated explicit activation of the renderability extensions if they're supported. If the language isn't strong enough then perhaps this sentence:

"Implementations supporting float rendering via this extension will implicitly enable..."

should also change "will" to "must".

Reply:
I propose going straight to:
Upon activation of this extension, implementations supporting WEBGL_color_buffer_float shall implicitly enable it.
NB: This requirement maintains the historical behavior prior to the differentiation of float renderability from float textures, so as to not break existing content.

Reply:
Sounds good to me.

Reply:
Addressed in the spec and leave the test as is.